File: tests/codegen/snapshots/relay/typescript/relay_nested_nodes.ts
type RelayNestedNodesResultPostAuthor = {
    id: string
    name: string
}

type RelayNestedNodesResultPost = {
    id: string
    title: string
    author: RelayNestedNodesResultPostAuthor
}

type RelayNestedNodesResult = {
    post: RelayNestedNodesResultPost
}

File: tests/codegen/snapshots/relay/python/relay_nested_nodes.py
from typing_extensions import TypedDict

class RelayNestedNodesResultPostAuthor(TypedDict):
    id: str
    name: str

class RelayNestedNodesResultPost(TypedDict):
    id: str
    title: str
    author: RelayNestedNodesResultPostAuthor

class RelayNestedNodesResult(TypedDict):
    post: RelayNestedNodesResultPost
